src/core/value_IOU.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const Value = require('./value').Value;
+const GlobalBigNumber = require('bignumber.js');
+const BigNumber = GlobalBigNumber.another({
+  ROUNDING_MODE: GlobalBigNumber.ROUND_HALF_UP,
+  DECIMAL_PLACES: 40
+});
+
+function Value_IOU(value) {
+  Value.call(this, value);
+}
+
+Value_IOU.prototype = Object.create(Value.prototype);
+
+Value_IOU.prototype.constructor = Value_IOU;
+
+Value_IOU.fromXRPValue = function(XRPvalue, bi_xns_unit) {
+  const newV = new Value_IOU(XRPvalue);
+  newV._XRP = true;
+  newV._bi_xns_unit = bi_xns_unit;
+  return newV;
+};
+
+Value_IOU.prototype.multiply = function(multiplicand) {
+  let mult = multiplicand;
+  if (mult._XRP) {
```

Comment:
I know this is more or less a result of the way rippled does math, but it's still really strange to have an `_XRP` flag in an IOU value class. Maybe instead of `fromXRPValue` we can just provide `multiplyByXRP` and `divideByXRP`?